.github/workflows/release.yaml

```diff
@@ -0,0 +1,71 @@
+# Prefect Operator release workflow.
+name: Release
+
+on:
```

Comment:
```suggestion
"on":
```

Reply:
Down for this, any reason why though? The [docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#on) don't seem to do it this way but maybe YAML keywords bite us sometimes?

Reply:
i think yamllint with throw an error. 'on' is a truthy value in yaml iirc. it won't break anything either way

Reply:
oh interesting, yeah I found https://github.com/adrienverge/yamllint/issues/430. weird one - will change

Reply:
Updated in 0ccd597875298df.